File: Aula 14 - Resiliência e persistencia/logico.py/nivel2.py
with open("log_sistema.txt", "w") as arquivo_log:

    arquivo_log.write("Sistema iniciado com sucesso.\n")

try:

    print("Enviando arquivo...")

except Exception as erro:

    print("Erro ao enviar arquivo.")

finally:

    print("Rotina de persistência finalizada, recursos liberados.")

File: Aula 14 - Resiliência e persistencia/logico.java/nivel2.java
import java.io.FileWriter;
import java.io.IOException;

public class Nivel2 {

    public static void main(String[] args) {

        try {

            FileWriter arquivo = new FileWriter("log_sistema.txt");

            arquivo.write("Sistema iniciado com sucesso.");

            arquivo.close();

            System.out.println("Arquivo salvo.");

        }

        catch(IOException erro) {

            System.out.println("Erro ao salvar arquivo.");

        }

        finally {

            System.out.println("Rotina de persistência finalizada, recursos liberados.");

        }

    }
}